### ISSUE TITLE
Not quite an issue, but a texture offer-

### ISSUE THREAD

**Issue by Liamgrayskull** [Rener-py/SCB#2]
for the purposes of my personal modpack, I was considering creating custom textures for this mod to pass them over with my personal style. Once complete, would you be interested in looking it over and potentially integrating it into the mod?

**Comment by R3ner** [comment]
Of course! I'll be happy to take a look at it, feel free to send it to me ^^

**Comment by Liamgrayskull** [comment]
![image](https://github.com/Rener-py/SCB/assets/129421895/ab570181-6663-4336-bbfb-8ef82289d81d)
![image](https://github.com/Rener-py/SCB/assets/129421895/2c27017f-0496-4a1f-b60b-daf54181233d)

In my personal style, here are some versions of the ghost entity skins I was feeling- how do you feel about these?

**Comment by R3ner** [comment]
I like them, I will put them through a test phase and try to make them available as an option

**Comment by Liamgrayskull** [comment]
Alright, thank you! I plan on redoing the rest of the textures as well, although I'm not quite sure how to go about texturing the ghost armor yet-
